SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -136,7 +163,15 @@ ln -s %{_datadir}/cloud-hypervisor/vmlinux.bin %{buildroot}/lib/modules/%{name}/
 /lib/modules/%{name}/vmlinux
 %endif
 
+%files devel
+%defattr(-,root,root)
+/lib/modules/%{uname_r}/build
+%{_prefix}/src/linux-headers-%{uname_r}
+
 %changelog
+* Wed Apr 5 2023 Chris Co <chrco@microsoft.com> - 5.15.98.mshv1-2
```

Comment:
Let's use Dallas here?

Reply:
This was originally Chris' change

Reply:
I'm good with either way, and I appreciate the thoughtfulness. I will leave it to @dallasd1 since this is as much your changes as it is mine. If anything, I only provided some initial commits.